services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/diagnostic_insights/__tests__/locked_page.test.tsx

```diff
@@ -35,7 +35,7 @@ describe('LockedPage Component', () => {
       expect(container).toMatchSnapshot();
     });
 
-    test('renders "Select school" variation when no valid school is selected', () => {
+    test('renders "Email us" variation when alternative school is selected', () => {
```

Comment:
Should we be adding to this test, rather than replacing it entirely? It feels like it still makes sense to test the other state here.